3rdparty/ymfm/src/ymfm_fm.h

```diff
@@ -33,6 +33,8 @@
 
 #pragma once
 
+#define DEBUG_LOG_WAVFILES (0)
+
```

Comment:
This header is included by the public headers ymfm_opl.h, ymfm_opn.h, etc. so this macro leaks downstream.  Please use some kind of project prefix for it to avoid possibility of conflicts, ans preprocessor macros are inherently unscoped (e.g. make it `YMFM_DEBUG_LOG_WAVFILES` or something).